orc8r/cloud/go/obsidian/access/rest_middleware.go

```diff
@@ -50,7 +50,7 @@ func Middleware(next echo.HandlerFunc) echo.HandlerFunc {
 		}
 
 		perms := getRequestedPermissions(req, decorate)
-		staticReadOnly := perms == accessprotos.AccessControl_READ && strings.HasPrefix(c.Path(), obsidian.StaticURLPrefix)
+		staticReadOnly := perms == accessprotos.AccessControl_READ && strings.HasPrefix(c.Path(), obsidian.StaticURLPrefixLegacy)
```

Comment:
This is probably hard to sanity check locally because we auto-permit requests if they come from localhost. Here's what you want

```
isStatic := strings.HasPrefix(c.Path(), obsidian.StaticURLPrefix) || strings.HasPrefix(c.Path(), obsidian.StaticURLPrefixLegacy)
isStaticReadOnly := isStatic && perms == ...
```